client/js/components/statement/assessmentTable/DpFragmentsSwitcher.vue

```diff
@@ -54,7 +54,7 @@ statements and fragments is now in DpAssessmentTableCard.vue
 
 <script>
 import { mapGetters, mapState } from 'vuex'
-import { DpSwitcher } from '@demos-europe/demosplan-ui'
+import { DpSwitcher } from './DpSwitcher'
```

Comment:
Actually i meant to move it _inside_ DpFragmentsSwitcher.vue.

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/2169/commits/98d20583e3012ef3e7021f0eb95fdea06ae459ca